### PR TITLE
Add System Commands

### PR DESCRIPTION
**Pull request by onesounds**
https://www.mirazon.com/shell-commands-for-windows-10-navigating-common-folders-in-windows/

**Added List**
- Open Recycle Bin
- Indexing Option

**ETC**
- I picked list that usually can't open by flow.
- These folders have different names depending on the language.
- It is necessary to search both English and native language, but it is currently difficult to implement, so it is recommended to use subtitle in native language and Title in English.

### CODE REVIEW

File: Plugins/Flow.Launcher.Plugin.Sys/Languages/ko.xaml [hunk 1]
```diff
@@ -15,6 +18,17 @@
...
+    <system:String x:Key="flowlauncher_plugin_sys_openmycomputer">내 PC</system:String>
+    <system:String x:Key="flowlauncher_plugin_sys_openmydesktop">바탕 화면</system:String>
+    <system:String x:Key="flowlauncher_plugin_sys_libraries">라이브러리</system:String>
+    <system:String x:Key="flowlauncher_plugin_sys_explorer">파일 탐색기</system:String>
+    <system:String x:Key="flowlauncher_plugin_sys_pictures">사진</system:String>
+    <system:String x:Key="flowlauncher_plugin_sys_music">음악</system:String>
+    <system:String x:Key="flowlauncher_plugin_sys_videos">비디오</system:String>
+    <system:String x:Key="flowlauncher_plugin_sys_downloads">다운로드</system:String>
+    <system:String x:Key="flowlauncher_plugin_sys_documents">문서</system:String>
+    <system:String x:Key="flowlauncher_plugin_sys_indexoption">색인 옵션</system:String>
```
Comment (taooceros) [inline Plugins/Flow.Launcher.Plugin.Sys/Languages/ko.xaml:31]: You may need to copy this back to crowdin if we merge🤔.

File: Plugins/Flow.Launcher.Plugin.Sys/Main.cs [hunk 2]
```diff
@@ -192,11 +192,161 @@ private List<Result> Commands()
...
+                        return true;
+                    }
+                },
+                new Result
                 {
                     Title = "Empty Recycle Bin",
                     SubTitle = context.API.GetTranslation("flowlauncher_plugin_sys_emptyrecyclebin"),
                     IcoPath = "Images\\recyclebin.png",
-                    Glyph = new GlyphInfo (FontFamily:"/Resources/#Segoe Fluent Icons", Glyph:"\xe74d"),
+                    Glyph = new GlyphInfo (FontFamily:"/Resources/#Segoe Fluent Icons", Glyph:"\xe896"),
```
Comment (taooceros) [inline Plugins/Flow.Launcher.Plugin.Sys/Main.cs:349]: Is this intended? `e896` is the download icon? 
image

**Review by taooceros — CHANGES_REQUESTED** [review]

File: Plugins/Flow.Launcher.Plugin.Sys/Main.cs (hunk 2 above)
Comment (onesounds) [reply Plugins/Flow.Launcher.Plugin.Sys/Main.cs]: mistake. Fixed e74d from e896.

File: Plugins/Flow.Launcher.Plugin.Sys/Languages/ko.xaml (hunk 1 above)
Comment (onesounds) [reply Plugins/Flow.Launcher.Plugin.Sys/Languages/ko.xaml]: ok.

**Comment by onesounds** [comment]
Well, come to think of it, only works with explorer. Should we set Total Commander or Dopus?

File: Plugins/Flow.Launcher.Plugin.Sys/Main.cs [hunk 3]
```diff
@@ -192,6 +192,156 @@ private List<Result> Commands()
...
+                    IcoPath = "Images\\explorer.png",
+                    Glyph = new GlyphInfo (FontFamily:"/Resources/#Segoe Fluent Icons", Glyph:"\xec50"),
+                    Action = c =>
+                    {
+                        {
+                            System.Diagnostics.Process.Start("explorer.exe", "");
+                        }
+
+                        return true;
+                    }
```
Comment (taooceros) [inline Plugins/Flow.Launcher.Plugin.Sys/Main.cs:207]: is this one needed? Isn't the program plugin already indexed the explorer?

File: Plugins/Flow.Launcher.Plugin.Sys/Main.cs [hunk 4]
```diff
@@ -192,6 +192,156 @@ private List<Result> Commands()
...
+                new Result
+                {
+                    Title = "My PC",
+                    SubTitle = context.API.GetTranslation("flowlauncher_plugin_sys_openmypc"),
+                    IcoPath = "Images\\mycomputer.png",
+                    Glyph = new GlyphInfo (FontFamily:"/Resources/#Segoe Fluent Icons", Glyph:"\xe7f4"),
+                    Action = c =>
+                    {
+                        {
+                            context.API.OpenDirectory("shell:mycomputerfolder");
```
Comment (taooceros) [inline Plugins/Flow.Launcher.Plugin.Sys/Main.cs:218]: this kind of link doesn't really got supported by 3rd explorer? Is the one you are using support it?

File: Plugins/Flow.Launcher.Plugin.Sys/Main.cs (hunk 3 above)
Comment (onesounds) [reply Plugins/Flow.Launcher.Plugin.Sys/Main.cs]: We don't need it. removed.

File: Plugins/Flow.Launcher.Plugin.Sys/Main.cs (hunk 4 above)
Comment (onesounds) [reply Plugins/Flow.Launcher.Plugin.Sys/Main.cs]: yeah, It has trouble with 3rd party file manager. I've tried a few things to work with third-party managers, but there's been a bit of a complicated problem. so I just simply remain only 'Open Recycle Bin' and 'Index Option'.

File: Plugins/Flow.Launcher.Plugin.Sys/Languages/en.xaml [hunk 5]
```diff
@@ -17,6 +17,17 @@
...
+    <system:String x:Key="flowlauncher_plugin_sys_openrecyclebin">Open recycle bin</system:String>
+    <system:String x:Key="flowlauncher_plugin_sys_openmypc">My PC</system:String>
+    <system:String x:Key="flowlauncher_plugin_sys_openmydesktop">Desktop</system:String>
+    <system:String x:Key="flowlauncher_plugin_sys_libraries">Libraries</system:String>
+    <system:String x:Key="flowlauncher_plugin_sys_explorer">Explorer</system:String>
+    <system:String x:Key="flowlauncher_plugin_sys_pictures">Pictures</system:String>
+    <system:String x:Key="flowlauncher_plugin_sys_music">Music</system:String>
+    <system:String x:Key="flowlauncher_plugin_sys_videos">Videos</system:String>
+    <system:String x:Key="flowlauncher_plugin_sys_downloads">Downloads</system:String>
+    <system:String x:Key="flowlauncher_plugin_sys_documents">Documents</system:String>
```
Comment (jjw24) [inline Plugins/Flow.Launcher.Plugin.Sys/Languages/en.xaml:29]:
```suggestion
```

**Review by jjw24 — APPROVED** [review]

**Pull request merged**